openmetadata-ui/src/main/resources/ui/src/utils/DataAssetsHeader.utils.tsx

```diff
@@ -422,6 +427,7 @@ export const getDataAssetsHeaderInfo = (
         <>
           {apiCollection.endpointURL && (
             <ExtraInfoLink
+              newTab
```

Comment:
@Sachin-chaurasiya can you confirm if `apiCollection.endpointURL` will be external or internal?

Reply:
Yes,@chirag-madlani that will be the external link.